addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -1992,6 +1992,47 @@ export class OdooEditor extends EventTarget {
             this.toolbarShow();
         }
     }
+    /**
+     * If the mouse is hovering over one of the borders of a table cell element,
+     * return the side of that border ('left'|'top'|'right'|'bottom').
+     * Otherwise, return false.
+     *
+     * @private
+     * @param {MouseEvent} ev
+     * @returns {boolean}
+     */
+    _isHoveringTdBorder(ev) {
```

Comment:
This is used only once. I suggest to inline it at call site.

Reply:
It will be used in a second place in a further commit.